fsharp/deprecated.md

```diff
@@ -0,0 +1 @@
+These images were removed in favor of the [official .NET images](https://hub.docker.com/_/microsoft-dotnet) maintained by Microsoft.
```

Comment:
Is there a specific image over there that current https://hub.docker.com/_/fsharp users should transition to?
(Having trouble understanding the mapping myself, and I imagine existing users might too, but perhaps not.)

Reply:
Indeed, I feel like this type of clarification is something Microsoft and the .NET Foundation are always working on, what with .NET trying to be everything for everyone. Let me know if the amended commit helps!